src/applications/mhv-medical-records/hooks/useAcceleratedData.js

```diff
@@ -67,15 +72,19 @@ const useAcceleratedData = () => {
   );
 
   const isAccelerating = useMemo(
-    () => isAcceleratingAllergies || isAcceleratingVitals,
-    [isAcceleratingAllergies, isAcceleratingVitals],
+    () =>
+      isAcceleratingAllergies ||
+      isAcceleratingVitals ||
+      isAcceleratingLabsAndTests,
```

Comment:
QUESTION: Is there a use case in which we would want to turn of OH integration overall and still keep the new experience? (or am I borrowing trouble thinking like this?)